### PR TITLE
Thumbnail: reorder

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1363
[Problem] Controllers- Grid List - reorder
[Solution]
 - added example of Thumbnail reorder
 - added AppBar lock during drag Thumbnail item
 - modify css of dragged item

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/js/profile/mobile/widget/GridView.js [hunk 1]
```diff
@@ -758,6 +763,41 @@
...
+					borderSize = self._borderSize;
+
+				// set margin
+				for (i = 0; i < length; i++) {
+					elementStyle = listElements[i].style;
+					// all without last in raw should have right border
+					if (i % cols < cols - 1) {
+						elementStyle.marginRight = borderSize + "px";
+					} else {
+						elementStyle.marginRight = "0"
```
Comment (DohyungLim) [inline src/js/profile/mobile/widget/GridView.js]: add semicolon.

File: src/js/profile/mobile/widget/GridView.js [hunk 2]
```diff
@@ -87,6 +88,7 @@
 				MATRIX_REGEXP = /matrix\((.*), (.*), (.*), (.*), (.*), (.*)\)/,
 				DATA_ROLE = "data-role",
 				RESIZE_TIMEOUT = 300,
+				BORDER_SIZE = 16,
```
Comment (DohyungLim) [inline src/js/profile/mobile/widget/GridView.js:91]: is BORDER_SIZE fixed?

File: examples/mobile/UIComponents/components/thumbnail/index.html [hunk 3]
```diff
@@ -45,6 +45,11 @@ <h1>
 						Thumbnail grid + text 2 lines
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="grid-reorder.html">
+						Thumbnail grid reorder
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/thumbnail/index.html]: Thumbnail Grid Reorder.
It is better to match the capital letters.

**Review by DohyungLim — COMMENTED** [review]
3 comment.

File: src/js/profile/mobile/widget/GridView.js (hunk 2 above)
Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/GridView.js:91]: Yes. The border size is fixed in widget. I moved this value from line 778/818

File: src/js/profile/mobile/widget/GridView.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/GridView.js]: Done.

File: src/js/profile/mobile/widget/GridView.js [hunk 4]
```diff
@@ -758,6 +763,41 @@
 				return holder;
 			};
 
+			/**
+			 * Set the margins of each item
+			 * @method _setItemSize
```
Comment (HunseopJeong) [inline src/js/profile/mobile/widget/GridView.js]: _setItemMargin.

Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/GridView.js]: Done.

File: examples/mobile/UIComponents/components/thumbnail/index.html (hunk 3 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/thumbnail/index.html]: Done.

**Review by korneliakobiela — APPROVED** [review]
Looks good and works good

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Review by singa2000 — APPROVED** [review]
LGTM

**Pull request merged**